{"name": "Логин с неверным логином и паролем", "status": "passed", "start": 1716873529943, "stop": 1716873531701, "uuid": "848c8791-5a96-4960-bbd6-d6708238a417", "historyId": "9b6a07466893ade475fa64741e646ff6", "testCaseId": "9b6a07466893ade475fa64741e646ff6", "fullName": "tests.test_user_login.TestUserLogin#test_login_with_incorrect_login_and_password", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_user_login"}, {"name": "subSuite", "value": "TestUserLogin"}, {"name": "host", "value": "GPRS"}, {"name": "thread", "value": "16248-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_user_login"}]}